tests/cephadm/test_osd_config.py

```diff
@@ -0,0 +1,51 @@
+import os
+import re
+
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import CephadmOpsExecutionError, MonDaemonError
+from utility.log import Log
+
+log = Log(__name__)
+
+CEPH_LIB_DIR = "/var/lib/ceph"
+
+
+def run(ceph_cluster, **kw):
+    """
+    Check the Number of Mon nodes in OSD config file
+    """
+    installer = ceph_cluster.get_ceph_object("installer")
+    osd_nodes = ceph_cluster.get_nodes(role="osd")
+
+    # Get cluster FSID
+    fsid = CephAdm(installer).ceph.fsid()
```

Comment:
As we are using `cephadm` here, `fsid` won't be clean string. Here you need to use client node and `ceph` object to get the fsid.